### PR TITLE
execution classes in control plane convert flyte literals to python

### PR DESCRIPTION
**Pull request by cosmicBboy**
Signed-off-by: cosmicBboy <niels.bantilan@gmail.com>

# TL;DR

Fixes https://github.com/flyteorg/flyte/issues/974. The `inputs` and `outputs` properties of the `FlyteWorkflowExecution` and `FlyteNodeExecution` should now return python values instead of flyte literals

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [ ] Smoke tested
 - [X] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: flytekit/control_plane/nodes.py [hunk 1]
```diff
@@ -195,9 +200,13 @@ def inputs(self) -> Dict[str, Any]:
                         _common_utils.load_proto_from_file(_literals_pb2.LiteralMap, tmp_name)
                     )
 
-            # TODO: need to convert flyte literals to python types. For now just use literals
-            # self._inputs = TypeEngine.literal_map_to_kwargs(ctx=FlyteContext.current_context(), lm=input_map)
-            self._inputs = input_map
+            task_id = self.task_executions[0].id.task_id
+            task = FlyteTask.fetch(task_id.project, task_id.domain, task_id.name, task_id.version)
+            self._inputs = TypeEngine.literal_map_to_kwargs(
```
Comment (kumare3) [inline flytekit/control_plane/nodes.py:205]: its not required to guess the types always right? We can provide the actual definition and then guessing is not required

Comment (cosmicBboy) [reply flytekit/control_plane/nodes.py:205]: what's the actual definition?

File: flytekit/control_plane/nodes.py [hunk 2]
```diff
@@ -229,9 +240,14 @@ def outputs(self) -> Dict[str, Any]:
...
                         _common_utils.load_proto_from_file(_literals_pb2.LiteralMap, tmp_name)
                     )
-            # TODO: need to convert flyte literals to python types. For now just use literals
-            # self._outputs = TypeEngine.literal_map_to_kwargs(ctx=FlyteContext.current_context(), lm=output_map)
-            self._outputs = output_map
+
+            task_id = self.task_executions[0].id.task_id
+            task = FlyteTask.fetch(task_id.project, task_id.domain, task_id.name, task_id.version)
+            self._outputs = TypeEngine.literal_map_to_kwargs(
+                ctx=FlyteContext.current_context(),
```
[the remark this answers is not in the document]
Comment (cosmicBboy) [reply flytekit/control_plane/nodes.py]: changed to FlyteContextManager

**Review by wild-endeavor — APPROVED** [review]
